backend/src/test/java/com/htecgroup/skynest/service/impl/UserServiceImplTest.java

```diff
@@ -201,16 +199,16 @@ void listAllUsers() {
     userEntityList.add(enabledWorkerEntity);
     when(userRepository.findAll()).thenReturn(userEntityList);
 
-    List<UserDto> expectedDTOs =
+    List<UserResponse> expectedResponse =
         userEntityList.stream()
-            .map(e -> modelMapper.map(e, UserDto.class))
+            .map(e -> modelMapper.map(e, UserResponse.class))
             .collect(Collectors.toList());
 
-    List<UserDto> returnedDTOs = userService.listAllUsers();
+    List<UserResponse> returnedResponse = userService.listAllUsers();
 
-    Assertions.assertEquals(expectedDTOs.size(), returnedDTOs.size());
-    Assertions.assertEquals(expectedDTOs.get(0), returnedDTOs.get(0));
-    Assertions.assertEquals(expectedDTOs.get(0).getEmail(), returnedDTOs.get(0).getEmail());
+    Assertions.assertEquals(expectedResponse.size(), returnedResponse.size());
```

Comment:
There is a good code structure the @MetoMladenovski did in his PR, maybe you need to change that according to his changes when he merges the PR.

Reply:
Yeah, there will be lots of test refactoring. Is it fine if I do that in a separate PR?

Reply:
yeah sure